### ISSUE TITLE
Fetal neck mass

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7369]
**Preferred term label:**
Fetal neck mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormal growth (lump) of tissue in the neck region of a fetus. Masses may be simple cystic,, cystic with a few simple septations (fewer than three septations and less than 3-mm thick), cystic with multiple or thick septations (greater than three septations or septations 3-mm-thick or more), a mass that is than is less than 50% solid, and a mass that is 50% solid or more.
PMID:30422714

Ascertained by Fetal MRI and sonography

**Parent term (use hpo.jax.org/app)**
Fetal neck anomaly HP:0025667

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
